packages/marketplace/src/components/pages/apps/coming-soon.tsx

```diff
@@ -26,19 +30,39 @@ const comingSoonImagesList = [
   comingSoonImage9,
 ]
 
-const ComingSoonApps: React.FC = () => {
+export const handleSetCommingSoonAppSectionHeight = (
+  containerHeight: number,
+  setCommingSoonAppSectionHeight: React.Dispatch<React.SetStateAction<number>> | undefined,
+) => {
+  return () => {
+    if (setCommingSoonAppSectionHeight) {
+      setCommingSoonAppSectionHeight(containerHeight)
+    }
+  }
+}
+
+const ComingSoonApps: React.FC<CommingSoonAppsProps> = ({ setCommingSoonAppSectionHeight }) => {
```

Comment:
Is it worth wrapping either the component or the handleSetCommingSoonAppSectionHeight func in a React.memo / useMemo so we're not calculating the height constantly?